src/WebJobs.Script.WebHost/Security/Authorization/AuthUtility.cs

```diff
@@ -27,10 +27,10 @@ public static AuthorizationPolicy CreateFunctionPolicy(IEnumerable<string> schem
             return new AuthorizationPolicy(requirements, schemes);
         }
 
-        public static bool PrincipalHasAuthLevelClaim(ClaimsPrincipal principal, AuthorizationLevel requiredLevel, string keyName = null)
+        public static bool PrincipalHasAuthLevelClaim(ClaimsPrincipal principal, AuthorizationLevel requiredLevel, string keyName = null, bool isFunctionDisabled = false)
         {
             // If the required auth level is anonymous, the requirement is met
-            if (requiredLevel == AuthorizationLevel.Anonymous)
+            if (requiredLevel == AuthorizationLevel.Anonymous && !isFunctionDisabled)
```

Comment:
Can you explain these changes? Are you saying that the checked in code as it stands isn't handling disabled functions properly?

Reply:
This is for disabling "httpTrigger". This part was missed for in the .net core port.

Reply:
I don't see how checking of auth claims should be dependent on whether a function is disabled. This doesn't seem right.

Reply:
As discussed I reverted this code and created an new issue for @fabiocav:
Add new middleware to handle "dissabled" state for httpTrigger #212